modules/rosa-nodes-machine-pools-local-zones.adoc

```diff
@@ -0,0 +1,73 @@
+====
+.Prerequisites
+
+* ROSA is generally available in the parent region of choice. See the link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/locations/?pg=ln&cp=bn#GA[AWS generally available locations list] to determine the Local Zone available to specific AWS regions.
+* The ROSA cluster was initially built in an existing Amazon VPC (BYO-VPC).
+* The maximum transmission unit (MTU) for the ROSA cluster is set at 1200.
++
+[IMPORTANT]
+====
+Generally, the Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300. See link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation. 
+The cluster network MTU must always be less than the EC2 MTU to account for the overhead. The specific overhead is determined by your network plugin, for example:
+
+- OVN-Kubernetes: `100 bytes`
+- OpenShift SDN: `50 bytes`
+
+The network plugin could provide additional features that may also decrease the MTU. Check the documentation for additional information.
+
+====
+* The AWS account has link:https://docs.aws.amazon.com/local-zones/latest/ug/getting-started.html#getting-started-find-local-zone[Local Zones enabled].
+* The AWS account has a link:https://docs.aws.amazon.com/local-zones/latest/ug/getting-started.html#getting-started-create-local-zone-subnet[Local Zone subnet] for the same VPC as the cluster.
+* The AWS account has a subnet that is associated with a routing table that has a route to a NAT gateway.
+* The AWS account has the tag `kubernetes.io/cluster/<infra_id>: shared' on the associated subnet.
+
+.Procedure
+
+. Create a machine pool on the cluster by running the following ROSA CLI command.
++
+[source,terminal]
+----
+$ rosa create machinepool -c <cluster-name> -i 
+----
++
+. Add the subnet and instance type for the machine pool in ROSA CLI. After several minutes, the cluster will provision the nodes.
++
+[source, terminal]
+----
+I: Enabling interactive mode <1>
+? Machine pool name: xx-lz-xx <2>
+? Create multi-AZ machine pool: No <3>
+? Select subnet for a single AZ machine pool (optional): Yes <4>
+? Subnet ID: subnet-<a> (region-info) <5>
+? Enable autoscaling (optional): No <6>
+? Replicas: 2 <7>
+I: Fetching instance types <8>
+----
++
+
+<1> Enables interactive mode.
+<2> Names the machine pool. This is limited to alphanumeric and a maximum length of 30 characters.
+<3> Set this option to no.
+<4> Set this option to yes.
+<5> Selects a subnet ID from the list.
+<6> Select yes to enable autoscaling or no to disable autoscaling.
+<7> Selects the number of machines for the machine pool. This number can be anywhere from 1 - 180.
+<8> Selects an instance type from the list. Only instance types that are supported in the selected Local Zone will appear.
++
+
+. Provide the subnet ID to provision the machine pool in the Local Zone.
```

Comment:
Is this part of the previous step, or a command-line program flow? Not sure where to provide it, otherwise.

Reply:
No, it is not part of the previous step, and no, it is not a command-line program flow. This is exactly how it needs to be written.

Reply:
🤷 I think that it's confusing, but I'm just one reviewer. The step doesn't say where I'm providing the ID.